### ISSUE TITLE
Will this be renamed to lms_mw eventually? or will it stay as it is? (what's correct?)

### ISSUE THREAD

**Issue by gerardodiego** [ifarup/ciefunctions#187]
https://github.com/ifarup/ciefunctions/blob/b14d120cfa7fb6474419af04ff3f37b2bb22d616/tc1_97/plot.py#L228

**Comment by janhenrikwold** [comment]
The naming rule (template) I introduced/used in the naming of the function plots was that functions plotting two dimensional chromaticity diagrams should be named by the two coordinate symbols (that is, the abscissa symbol followed by the ordinate symbol, for instance 'ls' and 'lm') followed by  underscore and an abbreviation for the type of diagram (for instance '_mb' for Boynton-MacLeod, and '_mw' for Maxwellian.  Thus 'lm_mw' names a Maxwellian lm diagram and 'ls_mb' names a MacLeod-Boynton ls diagram. When Ivar changed the name of the function plotting the MacLeod-Boynton ls diagrams from ls_mb into lms_mb, the name no longer accord with this naming rule. 

In my opinion I would therefore prefer to go back to the name 'ls_mb' for this function, and, accordingly, keep the name 'lm_mw' in case of the function plotting the Maxwellian lm diagrams. 

(In the naming rules that I have introduced/used all through the program/code, names like 'lms_mb' and 'lms_mw' are names for lists of all three chromaticity coordinates (also the one not shown in the associated chromaticity diagram. Thus I fear the renaming made by Ivar from lm_mb to lms_mb might appear as somewhat confusing.)

**Comment by ifarup** [comment]
Det er allerede reversert. Endringen var basert på en misforståelse.

17. des. 2018 17:37 skrev Jan Henrik Wold <notifications@github.com>:

The naming rule (template) I introduced/used in the naming of the function plots was that functions plotting two dimensional chromaticity diagrams should be named by the two coordinate symbols (that is, the abscissa symbol followed by the ordinate symbol, for instance 'ls' and 'lm') followed by underscore and an abbreviation for the type of diagram (for instance '_mb' for Boynton-MacLeod, and '_mw' for Maxwellian. Thus 'lm_mw' names a Maxwellian lm diagram and 'ls_mb' names a MacLeod-Boynton ls diagram. When Ivar changed the name of the function plotting the MacLeod-Boynton ls diagrams from ls_mb into lms_mb, the name no longer accord with this naming rule.

In my opinion I would therefore prefer to go back to the name 'ls_mb' for this function, and, accordingly, keep the name 'lm_mw' in case of the function plotting the Maxwellian lm diagrams.

(In the naming rules that I have introduced/used all through the program/code, names like 'lms_mb' and 'lms_mw' are names for lists of all three chromaticity coordinates (also the one not shown in the associated chromaticity diagram. Thus I fear the renaming made by Ivar from lm_mb to lms_mb might appear as somewhat confusing.)

-
